src/views/Analysis/Overview/AnalysisOverview.tsx

```diff
@@ -0,0 +1,120 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { AppState, FinishedGameState } from '../../../state/AppState';
+import { BigBoard } from '../../../components/Board/BigBoard/BigBoard';
+import {
+  Card,
+  CardPrimaryAction,
+  CardActions,
+  CardActionButton,
+} from '@rmwc/card';
+import { Typography } from '@rmwc/typography';
+import { ListDivider } from '@rmwc/list';
+import { Icon } from '@rmwc/icon';
+import styles from './AnalysisOverview.module.css';
+import appRoutes from '../../../routes/routes';
+import { Link } from 'react-router-dom';
+
+interface AnalysisOverviewProps {
+  finishedGames: FinishedGameState[];
+}
+
+function getGameSummary(winner: 'X' | 'O' | null, moves: number) {
+  if (winner) {
+    return (
+      <>
+        <Icon
+          icon={{
+            icon: winner.toLowerCase(),
+            size: 'medium',
+          }}
+          className={styles.winnerIcon}
+        />
+        won after {moves} moves.
+      </>
+    );
+  }
+
+  return <>{moves} moves resulted in draw.</>;
+}
+
+function getLink(game: FinishedGameState) {
+  return {
+    tag: Link,
+    to: appRoutes.AnalysisParam.replace(
+      ':param',
```

Comment:
Is this the way to go?

Reply:
Yes. Alternatively you can convert AnalysisParam to a function.

Reply:
Ok. I'll give the function approach a try.